addons/account_check_printing/tests/test_print_check.py

```diff
@@ -13,14 +13,21 @@ class TestPrintCheck(AccountTestInvoicingCommon):
     def setUpClass(cls, chart_template_ref=None):
         super().setUpClass(chart_template_ref=chart_template_ref)
 
-        cls.payment_method_check = cls.env.ref("account_check_printing.account_payment_method_check")
+        bank_journal_id = cls.company_data['default_journal_bank']
 
-        cls.company_data['default_journal_bank'].write({
-            'outbound_payment_method_ids': [(6, 0, (
-                cls.env.ref('account.account_payment_method_manual_out').id,
-                cls.payment_method_check.id,
-            ))],
+        #todo fix these names
+        cls.payment_method_check = cls.env.ref('account_check_printing.account_payment_method_check')
+        cls.payment_method_check = cls.env['account.payment.method.line'].create({
+            'payment_method_id': cls.payment_method_check.id,
+            'journal_id': bank_journal_id.id
         })
+        payment_method_id = cls.env.ref('account.account_payment_method_manual_out')
+        cls.outbound_payment_method = cls.env['account.payment.method.line'].create({
+            'payment_method_id': payment_method_id.id,
+            'journal_id': bank_journal_id.id
+        })
+        bank_journal_id.outbound_payment_method_line_ids\
+            .filtered(lambda m: m.id not in (cls.outbound_payment_method.id, cls.payment_method_check.id)).unlink()
```

Comment:
Since `account_payment_method_check` is enabled by default, do you really need such kind of diff now?